docker/front/Dockerfile

```diff
@@ -6,6 +6,10 @@ ENV LC_ALL=C.UTF-8
 # NODE_ENV used to run front server on production
 ENV NODE_ENV=production
 
+# These variables are for independent from front/config.js
```

Comment:
Weird English

Reply:
Do these variables override ones in front/config.js, or do they serve as defaults when front/config.js doesn't have any variables set?

Reply:
Either.

Reply:
If front/config.js says that port is 4444 and the shell variable says PORT=5555, which value will actually be used?

Reply:
5555

Reply:
```suggestion
# Overrides for variables in front/config.js
```